fbssdc/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to FBSSDC
```

Comment:
Is this and other similar files necessary given that it's now just a folder and not a separate repo?

Reply:
It's not necessary, I'll remove such files after the merge